### DISCUSSION TITLE
Running migration in production

### DISCUSSION THREAD

**Discussion by fredmaiaarantes** [adonisjs/core#1320] [Help] [answered]
Hi there,
I'm having an issue when running Adonis migrations in production. The process should run without depending on anyone to confirm the running. By default, Adonis asks "You are in production environment. Want to continue running migrations?" but I need to force the command to run.

**I'm using V5** but I saw that on version 4 there was a -f flag to do this. I tried in Adonis 5 but it didn't work and I could not find an answer on the docs. **How can I force the migrations to run in production?**

This is what I found on V4: https://adonisjs.com/docs/4.0/migrations#_migrations_commands

<img width="750" alt="Screenshot 2020-07-24 at 15 17 06" src="https://user-images.githubusercontent.com/559305/88400796-c0c87900-cdc0-11ea-914d-1b901ed672dc.png">

Thanks.

**Comment by thetutlage** [comment, marked as answer]
For commands `--help` is the real docs. `node ace migration:run --help`

```
Run pending migrations

Usage: migration:run 

Flags
  --connection string   Define a custom database connection
  --force boolean       Explictly force to run migrations in production
  --dry-run boolean     Print SQL queries, instead of running the migrations
```

**Reply by fredmaiaarantes** [reply]
Thanks, @thetutlage! I must get used to using --help.

I just tested and it worked fine. Cheers.

**Reply by ezehlivinus** [reply]
> Thanks, @thetutlage! I must get used to using --help.
> 
> I just tested and it worked fine. Cheers.

@fredmaiaarantes I tried running same command it failed with the following error:

![image](https://user-images.githubusercontent.com/31221649/112304626-fef2a100-8c9d-11eb-9f47-b376467862ec.png)

cc:
@thetutlage

**Reply by ivan-prats** [reply]
@ezehlivinus do you perhaps have the package `@adonisjs/assembler` in devPackages?

If so: try putting it alongside the rest of production packages and try again. Some hosting providers clean the devPackages, and you need the assembler to run `node ace` commands

**Reply by ezehlivinus** [reply]
Okay...Naturally, when installing adonis, it put `@adonisjs/assembler` under devDependencies may be I have to remove it then install it to move it into dependencies....and try it again.

I will let you know the result...
